### PR TITLE
Added Stocks Metrics

### PR DESCRIPTION
**Pull request by lilyfrancis**
This is a mobile web application that collects information from a financial API to provide comprehensive data for stocks. Built with React and Redux. In this project I added the following:

- Installed basic dependencies and linters.
- Styled the application using plain CSS and React Bootstrap.
- Fetched data from the API when the application starts.
- Dispatched an action to store the selected data in the Redux store.
- Used useSelector(), Redux Hook, to select the state slices and render countries and country details in corresponding routes.
- Write unit tests with Jest and React Testing Library

### CODE REVIEW

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by arslanbisharat** [comment]
Hi, 
apologize for the inconvenience. 
It seems like the inline comments I made didn't reach out to you. Please allow me a few minutes to go through the Project and submit those again as you have made some changes.

File: src/components/__tests__/company.test.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+test('Each company page render', () => {
+  const home = renderer
+    .create(
+      <Provider store={store}>
+        <Route>
+          <CompanyTop symbol="AAPL" />
+        </Route>
+      </Provider>,
+    )
+    .toJSON();
```
Comment (arslanbisharat) [inline src/components/__tests__/company.test.js:103]: - [ ] Tests are failing in your project. Let's make sure you add more tests cases in your project and all of them are saved. 

Capture

**Review by DeliceLydia — APPROVED** [review]
# Congratulations! 🎉🎉

## Hi @lilyfrancis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional
- [ ] Please fix your screenshots link in the README currently is not showing

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,111 @@
-# stocks-metrics-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Stock Metrics APP
+
+>This is a mobile web application that collects information from an API to provide comprehensive stocks data.
+
+![Image](./images/cover.png)
```
Comment (arslanbisharat) [inline README.md]: - [ ] This link is broken that's why the image is not rendering. Please have a look at it.

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+[Nelson Sakwa](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) whose design template was used for this project.
+
+## 👊🏾 Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (arslanbisharat) [inline readme.md]: - [ ] You have duplicated the Readme file. Please make sure you have only one.

File: src/components/Home.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
```
Comment (arslanbisharat) [inline src/components/Home.js:1]: - [ ] You have only one page. As per guidelines, you need to have two pages.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]

**Pull request merged**